src/Menu.java

```diff
@@ -1,12 +1,51 @@
-public class Menu {
+import island.FirstInitialize;
+import island.Plants;
+import island.animals.Animals;
+import island.map.Island;
+
+
+
```

Comment:
in this class should be method like run/start where we prepare some eexecutor service with all animals(threads) in animals we need to add logic for moving/eating and add to run method(implemented from runnable)